Ashsish

```diff
@@ -0,0 +1 @@
+
```

Comment:
import java.util.HashMap;
import java.util.Map;

class Solution {
    public boolean canBeEqual(int[] target, int[] arr) {
        if (target.length != arr.length) {
            return false;
        }
        
        Map<Integer, Integer> countMap = new HashMap<>();
        
        // Count frequency of elements in target array
        for (int num : target) {
            countMap.put(num, countMap.getOrDefault(num, 0) + 1);
        }
        
        // Count frequency of elements in arr array
        for (int num : arr) {
            if (!countMap.containsKey(num)) {
                return false; // If arr has an element not in target
            }
            countMap.put(num, countMap.get(num) - 1);
            if (countMap.get(num) == 0) {
                countMap.remove(num);
            }
        }
        
        // Check if all counts are zero
        return countMap.isEmpty();
    }
}